packages/components-js/tests/e2e/specs/checkbox.e2e.ts

```diff
@@ -0,0 +1,487 @@
+import { expect, type Locator, test, type Page } from '@playwright/test';
+import {
+  addEventListener,
+  getActiveElementTagName,
+  getElementStyle,
+  getEventSummary,
+  getFormDataValue,
+  getLifecycleStatus,
+  getProperty,
+  hasFocus,
+  setContentWithDesignSystem,
+  setProperty,
+  skipInBrowsers,
+  waitForInputTransition,
```

Comment:
Why is this ` waitForInputTransition` needed? Should not be necessary when using web first assertions